readme.md

```diff
@@ -315,6 +315,7 @@ Other awesome open source projects that use Lighthouse.
 * **[lighthouse-cron](https://github.com/thearegee/lighthouse-cron)** - Cron multiple batch Lighthouse audits and emit results for sending to remote server.
 * **[lighthouse-gh-reporter](https://github.com/carlesnunez/lighthouse-gh-reporter)** - Run Lighthouse in CI and report back in a comment on your pull requests
 * **[lighthouse-hue](https://github.com/ebidel/lighthouse-hue)** - set the color of Philips Hue lights based on a Lighthouse score
+* **[lighthouse-jest-example](https://github.com/justinribeiro/lighthouse-jest-example)** - gathers performance metrics via Lighthouse and tests results with Jest; uses Puppeteer to start up come with Chrome with network emulation settings defined by WebPageTest.
```

Comment:
```suggestion
* **[lighthouse-jest-example](https://github.com/justinribeiro/lighthouse-jest-example)** - gather performance metrics via Lighthouse and assert results with Jest; uses Puppeteer to start Chrome with network emulation settings defined by WebPageTest.
```

Reply:
the `gather` bit is just since the other projects tend to say what the user can do with the project rather than what the project does, i.e. they're written with "With this project you can ________"

feel free to reject the `assert` change if you prefer without it :)

Reply:
:+1: I like more specific!